workshops/migrations/0060_published_help_text.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+import django.core.validators
+import re
+import django_countries.fields
+
+
+def add_address_to_online_events(apps, schema_editor):
+    """Set Event.address if empty for online events."""
+    Event = apps.get_model('workshops', 'Event')
+    Tag = apps.get_model('workshops', 'Tag')
+    online = Tag.objects.get(name='online')  # should be created via 0040_*
+
+    Event.objects.filter(tags__in=[online]) \
+                 .filter(models.Q(address=None) | models.Q(address='')) \
+                 .update(address='Internet')
```

Comment:
Also, 0040 was setting `Event.venue` to `Internet` but  this one is setting `Event.address` to `Internet`.  Since we need both to publish, we probably want an additional data migration that sets `venue` to `Internet` for all existing online events.